docs/docs/ref-10-glossary.md

```diff
@@ -189,5 +190,7 @@ type ReactComponent<TProps> = {
   props : TProps,
   render : () => ReactElement
 };
+
+type ReactStatelessComponent<TProps> = (TProps) => ReactElement;
```

Comment:
I'd leave it as it's the same as in the source:
https://github.com/facebook/react/blob/master/src/renderers/shared/reconciler/__tests__/ReactStatelessComponent-test.js